webpack.config.js

```diff
@@ -4,8 +4,7 @@ const MiniCssExtractPlugin = require("mini-css-extract-plugin");
 
 const config = {
   entry: {
-    index: path.resolve(__dirname, "./src/index.js"),
-    main: path.resolve(__dirname, "./src/main/main.js"),
+    main: path.resolve(__dirname, "./src/index.js"),
```

Comment:
It must be `main` can't it be `index` 😄 ?

Reply:
I'll check that tomorrow what's going on with that, because in my local project I delete main and left only index.js